### DISCUSSION TITLE
Why are cells formatted as time, returned as a Duration?

### DISCUSSION THREAD

**Discussion by pressing-buttons** [miachm/SODS#52] [General]
I was wondering why a cell formatted as a "time" value (e.g. 22:30:15 using `HH:MM:SS`) are returned as a `Duration` rather than a `LocalTime`? 

It's not a big deal to convert that into a LocalTime instance, but I find the choice of Duration a bit surprising.

**Comment by miachm** [comment]
Not reason, it should be LocalTime. Not sure why I thought Duration was the right class for this.
